cpp/src/dlpack/dlpack.cpp

```diff
@@ -134,11 +134,11 @@ std::unique_ptr<experimental::table> from_dlpack(
   CUDF_EXPECTS(tensor.ndim > 0 && tensor.ndim <= 2,
                "DLTensor must be 1D or 2D");
 
-  CUDF_EXPECTS(tensor.shape[0] > 0, "DLTensor first dim empty");
+  CUDF_EXPECTS(tensor.shape[0] >= 0, "DLTensor first dim should be of shape greater than or equal-to 0.");
```

Comment:
Made this change to let the column to be of empty(0) shape as well.

Reply:
may want to add a GTest for this case but will let a cpp reviewer advise :)

Reply:
Added one for review.